.github/workflows/tests.yaml

```
@@ -215,6 +215,52 @@ jobs:
           . venv\\Scripts\\activate
           pytest -vv --durations=20 --benchmark-disable tests/
 
+  tests-macos:
+    name: run / ${{ matrix.python-version }} / macOS
+    runs-on: macos-latest
+    timeout-minutes: 25
+    needs: tests-linux
+    strategy:
+      fail-fast: false
+      matrix:
+        # We only run on the latest supported version on Mac
+        python-version: [3.7]
```

```suggestion
        # We only run on the latest supported version on Mac
        python-version: [3.10]
```
If it's that latest version, shouldn't it be `3.10`?

I do agree though to only test one, max two Python version for MacOS.

---

Ah that should probably be `oldest`. I'm not sure I don't really mind and don't expect to find many actual regression so using 1 to free up runners is probably best.

Do you have a preference for either?

---

Maybe a slight preference towards `3.10` but there are good arguments for `3.7` too.

---

Benefit of using `3.7` is that we're likely to update it when we bump last supported version. Feel free to change it if you want to! But I kept it on `3.7` for now!